client/my-sites/site-settings/media-settings/index.jsx

```diff
@@ -0,0 +1,114 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import SectionHeader from 'components/section-header';
+import Card from 'components/card';
+import Button from 'components/button';
+import FormSelect from 'components/forms/form-select';
+import FormLabel from 'components/forms/form-label';
+import FormCheckbox from 'components/forms/form-checkbox';
+import JetpackModuleToggle from '../jetpack-module-toggle';
+import FormFieldset from 'components/forms/form-fieldset';
+import InfoPopover from 'components/info-popover';
+import { isModuleActive, getModule } from 'state/jetpack-settings/modules/selectors';
+
+const MediaSettings = ( props ) => {
+	return (
+		<div>
+			<SectionHeader label={ props.translate( 'Media' ) }>
+				<Button
+					compact
+					primary
+					onClick={ props.submitFormAndActivateCustomContentModule }
+					disabled={ props.fetchingSettings || props.submittingForm }>
+					{ props.submittingForm ? props.translate( 'Saving…' ) : props.translate( 'Save Settings' ) }
+				</Button>
+			</SectionHeader>
+			<Card className="media-settings__card site-settings">
+				<FormFieldset>
+					<div className="media-settings__fieldset-infopopover">
```

Comment:
It might be wise to make this its own component specifically for what we're using it for. This way we can style it in a slightly more global way.

We also should probably absolutely position it. I can do that when you make the component.